crypto/engine/eng_openssl.c

```diff
@@ -422,7 +422,11 @@ static EVP_PKEY *openssl_load_privkey(ENGINE *eng, const char *key_id,
     EVP_PKEY *key;
     fprintf(stderr, "(TEST_ENG_OPENSSL_PKEY)Loading Private key %s\n",
             key_id);
+# if defined(OPENSSL_SYS_WINDOWS)
+    in = BIO_new_file(key_id, "rb");
+# else
     in = BIO_new_file(key_id, "r");
+# endif
```

Comment:
Apologies, I haven't followed this discussion in detail, but is there any benefit in adding these Windows-specific branches? Can't all OS you want to support deal with `"rb"` here?

Reply:
Probably they can support `"rb"` I didn't think to check, just took what openssl did. That's certainly simpler, for most platforms the `"b"` is ignored after all.  Should I?

Reply:
I think it's worth a try. It's four probably useless #ifdef after all.